packages/bibtex/init.lua

```diff
@@ -133,6 +278,93 @@ To produce a full reference, use \autodoc:command{\reference{<key>}}.
 
 Currently, the only supported bibliography style is Chicago referencing, but other styles should be easy to implement.
 Adapt \code{packages/bibtex/styles/chicago.lua} as necessary.
+
+\smallskip
+\noindent
+\em{Notes on the supported BibTeX syntax}
+\novbreak
+
+\indent
+The BibTeX file format is a plain text format for bibliographies.
+
+The \code{@type\{…\}} syntax is used to specify an entry, where \code{type} is the type of the entry, and is case-insensitive.
+Any content outside entries is ignored.
+
+The \code{@preamble} and \code{@comment} special entries are ignored.
+The former is specific to TeX-based systems, and the latter is a comment (everything between the balanced braces is ignored).
+
+The \code{@string\{key=value\}} special entry is used to define a string or “abbreviation,” for use in other subsequent entries.
+
+The \code{@xdata} entry is used to define an entry that can be used as a reference in other entries.
+Such entries are not printed in the bibliography.
+Normally, they cannot be cited directly.
+In this implementation, a warning is raised if they are; but as they have no known type, their formatting is not well-defined, and might not be meaningful.
+
+Regular bibliography entries have the following syntax:
+
+\begin[type=autodoc:codeblock]{raw}
+@type{key,
+  field1 = value1,
+  field2 = value2,
+  …
+}
+\end{raw}
+
+The entry key is a unique identifier for the entry, and is case-sensitive.
+Entries consist of fields, which are key-value pairs.
+The field names are case-insensitive.
+Spaces and line breaks are not important, except for readability.
+On the contrary, commas are compulsory between any two fields of an entry.
+
+String values shall be enclosed in either double quotes or curly braces.
+The latter allows using quotes inside the string, while the former does not without escaping them with a backslash.
+
+When string values are not enclosed in quotes or braces, they must not contain any whitespace characters.
+The value is then considered to ba a refererence to an abbreviation previously defined in a \code{@string} entry.
```

Comment:
to ba > to be

Reply:
To “bah”, or not to “bah”, that is the question.

Reply:
Fixed and autosquashed